src/plugins/data_view_management/public/components/index_pattern_table/index_pattern_table.tsx

```diff
@@ -164,7 +164,9 @@ export const IndexPatternTable = ({
 
   chrome.docTitle.change(title);
 
-  const isRollup = new URLSearchParams(useLocation().search).get('type') === 'rollup';
+  const isRollup =
```

Comment:
Nit: since we do this multiple times, it might benefit from something like a `useIsRollup` hook.